provider/resources.go

```diff
@@ -3046,6 +3098,9 @@ func Provider() tfbridge.ProviderInfo {
 			"google_kms_crypto_key_version": {Tok: gcpDataSource(gcpKMS, "getKMSCryptoKeyVersion")},
 			"google_kms_secret_asymmetric":  {Tok: gcpDataSource(gcpKMS, "getKMSSecretAsymmetric")},
 			"google_kms_secret_ciphertext":  {Tok: gcpDataSource(gcpKMS, "getKMSSecretCiphertext")},
+
+			"google_logging_project_cmek_settings": {Tok: gcpDataSource(gcpLogging, "getProjectCMEKSettings")},
```

Comment:
[Traditionally we don't capitalize all letters in an initialism](https://github.com/pulumi/platform-providers-team/blob/main/playbooks/tf-provider-updating.md#reviewing-a-pr) but as long as we're not actively misspelling anything, and we already are saying `FeatureStore` instead of `Featurestore` above, the only way this would become a problem if we're ever going to automate this mapping.
```suggestion
			"google_logging_project_cmek_settings": {Tok: gcpDataSource(gcpLogging, "getProjectCmekSettings")},
```

Reply:
Ah, thanks for the pointer. I'll go ahead and update as suggested to avoid any further drift.